packages/plugins/src/hardhat/deployments.ts

```diff
@@ -200,30 +201,41 @@ export const deployChugSplashConfig = async (
 
   spinner.start('The deployment is being executed. This may take a moment.')
 
-  if (remoteExecution) {
-    await monitorTask(
+  // If executing locally, then startup executor with HRE provider and pass in canonical config
+  if (!remoteExecution) {
+    const executor = new ChugSplashExecutor()
+    executor.init()
+    executor.main(
       {
-        configPath,
-        silent: true,
+        privateKey:
+          '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80',
```

Comment:
I think we should send ETH from the current signer to the account that corresponds to the `0xac...` private key before calling `executor.main`. This is because users can define their own set of accounts on the Hardhat network, in which case the `0xac` account will hold 0 ETH and won't be able to execute the deployment. I think 1 ETH should be sufficient